kubernetes/config-maps/nginx-proxy/default.conf.j2

```diff
@@ -5,22 +5,22 @@ map $http_accept $extension {
     application/json json;
 }
 
-{% if nginx_proxy_ssl %}
+{% if not nginx_proxy_allow_http %}
 server {
     listen 80      default_server;
```

Comment:
We talked about it in person but I'll add it here to so that I don't forget. Just a minor change:

Please add a comments saying `HTTP -> HTTPS redirect`, `HTTP` and `HTTPS` before each part. this will make it a bit clearer.

Reply:
Done.